### DISCUSSION TITLE
Preventing delete from removing command

### DISCUSSION THREAD

**Discussion by LandonSchropp** [folke/noice.nvim#377] [Q&A]
I'm currently using inc-rename.nvim with noice.nvim. Since I have this bound to a shortcut (`<leader>cr`), the inc-rename.nvim command pops up automatically.

When renaming functions, I'm used to being able to hold the delete key until the old value is completely deleted, and then type in my new one. However, with noice.nvim, if I do this, I start to delete the `IncRename` command instead.

Is it possible to get noice.nvim's command prompt to stop deleting once the prompt is empty instead of continuing to delete the hidden command?

**Comment by folke** [comment]
No, since that is still just the cmdline. It just looks fancier

**Reply by LandonSchropp** [reply]
Gotcha, thanks!
